napi-inl.h

```diff
@@ -539,6 +539,10 @@ inline bool Value::IsNull() const {
   return Type() == napi_null;
 }
 
+inline bool Value::IsNullish() const {
+  return Type() == napi_undefined || Type() == napi_null;
```

Comment:
The compiler may optimize this since it is `const`, but lets store this in a variable to ensure it isn't called twice.